test/expected.txt

```diff
@@ -213,15 +213,16 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
-|| 
-|| 0 persons listed!
+|| 	1. Betsy Choo Tags: [secretive]
```

Comment:
Not shown here, but the input.txt says "find should be case-sensitive".
In this case, I think it should be "not case-sensitive".

Reply:
good catch!